### PR TITLE
updated it build status shape

### PR DESCRIPTION
**Pull request by mhenschke-atl**
My slightly blasé merges on Wednesday broke our integration tests. This should green them.

### CODE REVIEW

File: src/test/java/it/com/atlassian/bitbucket/jenkins/internal/status/BuildStatusPosterIT.java [hunk 1]
```diff
@@ -253,24 +251,27 @@ private static RequestPatternBuilder requestBody(RequestPatternBuilder requestPa
         assertNotNull(bitbucketRevisionAction);
         String refName = bitbucketRevisionAction.getBranchAsRefFormat();
         String jenkinsUrlAsString = jenkinsUrl.toExternalForm();
+        ItemGroup<?> parentProject = job.getParent();
+        String parentString = parentProject instanceof MultiBranchProject ? parentProject.getFullName() : job.getFullName();
```
Comment (alibk0rd) [inline src/test/java/it/com/atlassian/bitbucket/jenkins/internal/status/BuildStatusPosterIT.java:255]: nit: `parentString` is a bit vague and redundant (we can see the type is `String` ;-)
`parentName`? Just `parent`? Anything but `parentString` please!

Comment (mhenschke-atl) [reply src/test/java/it/com/atlassian/bitbucket/jenkins/internal/status/BuildStatusPosterIT.java:255]: Haha I was copying the form of `jenkinsUrlAsString` but it felt weird when I typed it. Lets go with `parentName` :)

Comment (mhenschke-atl) [inline src/test/java/it/com/atlassian/bitbucket/jenkins/internal/status/BuildStatusPosterIT.java:255]:
```suggestion
        String parent = parentProject instanceof MultiBranchProject ? parentProject.getFullName() : job.getFullName();
```

File: src/test/java/it/com/atlassian/bitbucket/jenkins/internal/status/BuildStatusPosterIT.java [hunk 2]
```diff
@@ -253,24 +251,27 @@ private static RequestPatternBuilder requestBody(RequestPatternBuilder requestPa
...
-                                       "\"serverIdentifier\":\"%s\"," +
                                        "\"state\":\"%s\"," +
                                        "\"url\":\"%s%sdisplay/redirect\"" +
                                        "}",
+                                        build.getId(),
                                         buildState.getDescriptiveText(build.getDisplayName(), build.getDurationString()),
-                                        build.getDuration(), job.getFullName(), job.getFullDisplayName(), refName,
-                                        build.getExternalizableId(), jenkinsUrlAsString, buildState, jenkinsUrlAsString,
-                                        build.getUrl())
+                                        build.getDuration(), job.getFullName(), job.getDisplayName(), parentString, refName,
```
Comment (mhenschke-atl) [inline src/test/java/it/com/atlassian/bitbucket/jenkins/internal/status/BuildStatusPosterIT.java:273]:
```suggestion
                                        build.getDuration(), job.getFullName(), job.getDisplayName(), parentName, refName,
```

**Pull request merged**